### PR TITLE
feat : Added Single video page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Added Single video page

- User can directly redirect to the selected video by its id.
- User can the other list of videos of selected category.
Screenshot (20)
Screenshot (21)

### CODE REVIEW

File: src/Routers.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ export const Routers = () =>{
             <Route path="likes" element={<Likes />}></Route>
             <Route path="login" element={<Login />}></Route>
             <Route path="signup" element={<SignUp />}></Route>
-            <Route path="explore" element={<Explore />}></Route>
+            <Route path="video" element={<Explore />}></Route>
             <Route path="history" element={<History />}></Route>
```
Comment (rahulyadav139) [inline src/Routers.js:11]: Suggestions: you can use self closing "Route" tag here.

File: src/Pages/Explore/Explore.js [hunk 2]
```diff
@@ -40,7 +41,7 @@ export const Explore = () =>{
                     </button>
                 )}
             </div>
-            <div className="video-container">
+            <div className="videos-container">
                 {sorted.map((video)=> 
                     <div class="card">
```
Comment (rahulyadav139) [inline src/Pages/Explore/Explore.js]: Suggestions: I think you forgot to write "key" attribute here.

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: Everything is great. I put some suggestions. Otherwise codes look great and readable. Great work.

File: src/Components/MustWatch/MustWatch.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    const  navigate = useNavigate()
+    const getVideos = videos.filter((item) => item.categoryName === getSingleVideo[0].categoryName)
+    const getSimilarCategoryVideos = getVideos.filter((item) => item._id !==_id)
+    return(
+        <>
+            { 
+                getSimilarCategoryVideos.map((video)=> 
+                    <div class="card">
+                        <>
+                            <img src={video.thumbnail} class="img-lg"/>
```
Comment (shivanigangadharan) [inline src/Components/MustWatch/MustWatch.js]: Add `alt` message for `img` tag

**Review by shivanigangadharan — APPROVED** [review]
Can work on suggestions given by Rahul, other than that code looks good. Do fix issues in deploy preview before merging :)

File: src/Components/MustWatch/MustWatch.js (hunk 3 above)
Comment (aiman-ansari) [reply src/Components/MustWatch/MustWatch.js]: sure I will add that

**Pull request merged**